### PR TITLE
Add marketing info and copy

### PR DESCRIPTION
**Pull request by seancolsen**
This PR adds some new resources allowing us to be more organized and consistent with our approach to marketing.

[Rendered copy](https://github.com/centerofci/mathesar-wiki/blob/marketing/marketing/copy.md)

### CODE REVIEW

**Comment by seancolsen** [comment]
- @kgodey in our chat you proposed: 

    > ...it would be nice to have the following standardized descriptions that we could pull for various things:
    >
    > - 1-sentence tagline
    > - 2 sentence description
    > - 1 paragraph description
    > - 3 paragraph description
    >
    > with versions for both technical audiences and general audiences

    That makes _eight_ different pieces of copy. I wanted to simplify that a bit, both for consistency across sites, and for ease of implementation.

- This PR proposes the following _three_ pieces of copy:

    - Technical tagline
    - Non-technical tagline
    - Technical description

- As far as a "non-technical description", we decided that the only two sites which need non-technical copy are https://mathesar.org and https://centerofci.org/mathesar/. I'd rather not mess with the copy there very much right now. For one thing, I think it's already quite good. And for another thing I don't think it's worth attempting to make it consistent, because it's useful to have more contextually-specific info in those places.

- In formulating the copy in this PR, I looked at all the copy on all our sites, as well as some competitor's sites.

- Additionally, I added some fundraising copy, that I plan to use on

    - OpenCollective
    - GitHub Sponsors
    - A new page on mathesar.org, to be implemented as part of the "integrate into website" task within https://github.com/centerofci/mathesar-website/issues/54

File: marketing/copy.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+tags: 
+editor: markdown
+dateCreated: 2023-02-23T00:00:00.000Z
+---
+
+## Tagline
+
+### Technical
+
+> Mathesar is an open source web application providing an intuitive spreadsheet-like interface to PostgreSQL databases.
```
Comment (kgodey) [inline marketing/copy.md:15]: @seancolsen I'm not a fan of "spreadsheet-like" as a descriptor for Mathesar for a few reasons:
- It completely ignores the data explorer, which I think is one of our best features and one of our biggest differentiators from competitors. Our tagline should describe that we support both data entry _and_ data analysis.
- We lack a lot of common features that spreadsheets have (because we're not actually focusing on making Mathesar like a spreadsheet), and I think people may be disappointed.

File: marketing/copy.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+### Non-technical
+
+> Mathesar is an application for managing structured data from an intuitive spreadsheet-like web interface.
```
Comment (kgodey) [inline marketing/copy.md:20]: This seems like it would be somewhat scary for non-technical users. I don't think they will know what "structured data" means. We should try to speak in the "voice of the customer". I recommend taking inspiration from this page: https://mathesar.org/non-technical-expert.html, which was based on Myra's research into non-technical users.

File: marketing/copy.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Mission
+
+> Our mission is to build a tool that lets people of all skill levels use data to solve complex problems.
+
+## Fundraising
+
+> `{mission}`
+>
+> As a non-profit project, Mathesar relies on donations to fund development. A small core team of paid engineers guide the direction of project and maintain its underlying architecture. Leveraging open source principles, the core team also amplifies its work by facilitating a broader community of volunteer contributors to help build the product.
```
Comment (kgodey) [inline marketing/copy.md:44]: "relies on donations" seems a bit disingenuous to me because we're currently grant-funded and plan to be funded through operating a SaaS. I'd make this more along the lines of "Your donations will help us do more."

File: marketing/copy.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+> Features such as foreign keys, data types, and schemas directly reflect the behavior of Postgres, allowing users of all skill levels to collaboratively work with the same normalized relational data that other applications might use too. You can model your data, alter its structure, quickly enter data, and even generate reports. Self-hosting Mathesar gives you control, ownership, and privacy of your data.
+
+### Non-technical
+
+_(varies depending on context)_
+
+
+## Mission
+
+> Our mission is to build a tool that lets people of all skill levels use data to solve complex problems.
```
Comment (kgodey) [inline marketing/copy.md:38]: I've got lots of thoughts on the mission, but I think it might make more sense for me to write a draft later than to try and critique this.

File: marketing/copy.md (hunk 1 above)
Comment (kgodey) [reply marketing/copy.md:15]: I'm actually not sure if "spreadsheet-like" is my issue here. I think we need more personality to our tagline. This is very dry. I like our Twitter / README ones better:

> An intuitive UI for managing data, for users of all technical skill levels. Built on Postgres.

> Mathesar is a straightforward open source tool that provides a spreadsheet-like interface to a PostgreSQL database.

File: marketing/copy.md (hunk 2 above)
Comment (kgodey) [reply marketing/copy.md:20]: Also see above note about personality.

**Review by kgodey — CHANGES_REQUESTED** [review]
@seancolsen This looks great overall, thanks for your work on this.

This is a good start for standard copy, we can always add more as we run into use cases. Your plan in your comment sounds fine, except that I'd rather not make any changes to the taglines in mathesar.org. The copy there is designed to all work together, and I don't want to change small pieces in isolation.

File: marketing/copy.md (hunk 3 above)
Comment (seancolsen) [reply marketing/copy.md]: > "relies on donations" seems a bit disingenuous

Good point. My intention here was to consider grants as a form of donation, but I agree that it's misleading as-written. I've pushed 331ee721e0313822680b55cf8155fc3671d75277 which I hope adds enough clarity.

> plan to be funded through operating a SaaS

The text after my changes won't be completely accurate if/once we launch a paid SaaS product, but I figure we can update the text as-needed.

File: marketing/copy.md (hunk 4 above)
Comment (seancolsen) [reply marketing/copy.md:38]: I'm trying to improve this text on our [GH sponsors page](https://github.com/sponsors/centerofci)

> Our mission is to build a tool to make **understanding and working with data easy for _everyone_.**

Specifically, the text _"to make understanding and working with"_ is kind of a mouthful. I find it hard to parse smoothly.

I also wanted to tie into CCI's mission better. But I'm fine with letting go of that.

We could try:

> Our mission is to build a tool that makes understanding and working with data easy for everyone.

or

> Our mission is to build a tool that makes it easy for everyone to understand and work with data.

or

> Our mission is to make it easy for everyone to understand and work with data.

Or we could remove the mission part entirely, just starting the "Fundraising" section off with _"As a non-profit project..."_

I don't feel too strongly about this piece of content.

**Comment by seancolsen** [comment]
@kgodey

- > I'd rather not make any changes to the taglines in mathesar.org

    Cool, that's fine with me.

- Regarding all your thoughts about the taglines, I think it would be the best use of both of our time if we hammered that out on a call.

**Review by kgodey — APPROVED** [review]
@seancolsen I'm going to merge this as-is and add a note in the copy page that we're still finalizing the copy. I'd like to have the marketing section to store links to publicity.
